webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -42,12 +50,61 @@ const CodeEditor = React.lazy(() => import('hew/CodeEditor'));
 
 const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }: Props = {}) => {
   const idPrefix = useId();
-  const { canModifyWorkspaceAgentUserGroup, canModifyWorkspaceCheckpointStorage } =
-    usePermissions();
+  const {
+    canModifyWorkspaceAgentUserGroup,
+    canModifyWorkspaceCheckpointStorage,
+    canSetWorkspaceNamespaceBindings,
+  } = usePermissions();
   const [form] = Form.useForm<FormInputs>();
   const useAgentUser = Form.useWatch('useAgentUser', form);
   const useAgentGroup = Form.useWatch('useAgentGroup', form);
   const useCheckpointStorage = Form.useWatch('useCheckpointStorage', form);
+  const resourceManagers = useAsync(async (canceller) => {
+    try {
+      const response = await getKubernetesResourceManagers(undefined, { signal: canceller.signal });
+      return response.names;
+    } catch (e) {
+      handleError(e, {
+        level: ErrorLevel.Error,
+        publicMessage: 'Failed to fetch Resource Managers.',
+        silent: false,
+        type: ErrorType.Server,
+      });
+      return NotLoaded;
+    }
+  }, []);
+
+  const isNonK8RMError = (e: unknown): boolean => {
+    if (e instanceof DetError && e.publicMessage?.includes('operation not supported')) {
+      return true;
+    }
+    return false;
```

Comment:
the boolean expression inside the if check is simple enough that we should just return it instead:
```suggestion
    return e instanceof DetError && e.publicMessage?.includes('operation not supported');
```

Reply:
this makes the return type boolean | undefined rather than just boolean

Reply:
You should be able to just add a `?? false` at the end then.

Reply:
Done!